SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+The following table outlines which versions of Fuego are actively supported with security updates. Please ensure that you are using a supported version to benefit from the latest patches and improvements.
+
+| Version | Supported            |
+| ------- | -------------------- |
+| 0.x.y (x being the latest version released)     | :white_check_mark: |
+| 0.x.y (x being NOT the latest version released)     | :x: |
```

Comment:
Eh, I don't really like making the promise of maintaining older versions until at least `v1`, but considering the we are changing a lot all the time. I could be convinced.

Reply:
The `:x:` is a red cross, meaning we won't support them ;)

Reply:
AH. Reading raw markup is hard

Reply:
Yes, and it is also important to understand with raw text. Fixed by also adding raw text.